x-pack/plugins/apm/common/elasticsearch_fieldnames.ts

```diff
@@ -113,8 +113,8 @@ export const METRICSET_NAME = 'metricset.name';
 export const LABEL_NAME = 'labels.name';
 
 export const HOST = 'host';
-export const HOST_NAME = 'host.hostname';
-export const HOSTNAME = 'host.name';
+export const HOST_HOSTNAME = 'host.hostname';
+export const HOST_NAME = 'host.name';
```

Comment:
Thanks for doing all these changes! 
After discussing this internally we've come to the conclusion that we should always only use `host.name` (aka `HOST_NAME`). 

Would it be possible for you to replace all usages of `'host.hostname'`/`HOST_HOSTNAME` with 'host.name'/`HOST_NAME`?

Thanks for the help and apologies for the late changes.

Reply:
Sure, I did that in my last commit.